test/graph/someSubscription.graphql

```diff
@@ -0,0 +1,10 @@
+query SomeSubscription {
```

Comment:
Aren't you expected to list the variables here too?

```
query SomeSubscription($teamId: string, $offset: number, $isPrivate: boolean) { ...
```

I guess that would make replacement harder though, since that would have to be removed since we aren't actually passing them along to the GraphQL parser.

http://graphql.org/learn/queries/#variables

Reply:
That's the part that is tricky. I couldn't find a way to use the parser to remove those. I've added a regex match and remove the match now. Take a look.